api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -180,14 +180,16 @@ private DbWorkspace lookupWorkspace(String workspaceNamespace) throws NotFoundEx
   }
 
   @Override
-  public ResponseEntity<EmptyResponse> createRuntime(String workspaceNamespace) {
+  public ResponseEntity<EmptyResponse> createRuntime(String workspaceNamespace, Runtime runtime) {
```

Comment:
Nice to have: can this endpoint accept the absence of a body Runtime payload? This would ensure that older UI clients don't break when this goes out.